### ISSUE TITLE
📚 Documentation: poor contrast focus indicators on buttons [a11y][TheA11y100]

### ISSUE THREAD

**Issue by GrahamTheDevRel** [appwrite/website#276]
## Issue
Focus indicators on buttons are barely visible, the contrast with the background should be improved.

!["Go to console button" focused with a barely visible focus indicator](https://github.com/appwrite/website/assets/121672086/06d4932f-3869-4322-9c55-4989026e7e18)

!["Explore All technologies" focused with a barely visible focus indicator](https://github.com/appwrite/website/assets/121672086/8cc43c82-cbaf-459a-a1b5-e25a2e7b2b70)

## Why it matters
People navigating via keyboard rely on focus indicators to see where they are on the page. Without these it is easy to get lost when tabbing through the page.

## Suggested fix
Focus indicators should have a contrast ratio with the background of at least 3:1.

Find a suitable colour using the WebAim colour contrast tool.

One colour that could work is #fd366ebb if needing to use opacity.

## Relevant Success Criterion WCAG
[WCAG focus visible](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html)
[WCAG focus appearance](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance.html)

## Additional Notes
Good first issue, needs fixing site-wide.

[Found on stream on 2023-10-25](https://www.twitch.tv/videos/1959992199)

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by Arindam200** [comment]
Can I work on this?

**Comment by Emmarie-Ahtunan** [comment]
Will you assign this one to me, please? Web Accessibility is my thing. Thanks.

**Comment by TorstenDittmann** [comment]
This needs some insight from the design team before I can assign someone 👍🏻

**Comment by eldadfux** [comment]
Starting today this repo is archived. Please raise any issues with the new Appwrite website on the main platform repo: https://github.com/appwrite/appwrite
